content/_data/changelogs/weekly.yml

```diff
@@ -19477,6 +19477,103 @@
   # pull: 7721 (PR title: Bump spotless-maven-plugin from 2.34.0 to 2.35.0)
   # pull: 7723 (PR title: Bump stapler.version from 1775.v3181508ea_21e to 1777.v7c6fe6d54a_0c)
 
+  - version: '2.396'
+    date: 2023-03-21
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 7718
+        authors:
+          - janfaracik
+          - timja
+        pr_title: Revamp icon legend as a modal
+        message: |-
+          Revamp icon legend as a modal.
+      - type: rfe
+        category: rfe
+        pull: 7732
+        authors:
+          - janfaracik
+        pr_title: Remove the `expand-button` component as it's no longer used
+        message: |-
+          Remove the <code>expandbutton</code> component as it's no longer used.
+      - type: rfe
+        category: rfe
+        pull: 7712
+        authors:
+          - janfaracik
+        pr_title: Refresh the design of the About Jenkins page
+        message: |-
+          Refresh the design of the About Jenkins page.
+      - type: bug
+        category: regression
+        pull: 7740
+        issue: 69489
+        authors:
+          - NotMyFault
+        pr_title: "[JENKINS-69489] - Hide restart checkbox if the controller doesn't\
+          \ support it"
+        message: |-
+          Hide <code>Restart Jenkins</code> checkbox in the update center if the controller doesn't support it.
```

Comment:
The phrase "Restart Jenkins" consists of English words, not code, so a `<code>` tag is not an appropriate form of emphasis. While having no emphasis would be consistent with the preceding changelog entry ("Refresh the design of the About Jenkins page"), adopting a consistent non-code form of emphasis (e.g., bold or italics) would likely be a better solution.